index.js

```diff
@@ -0,0 +1,44 @@
+const express = require("express");
+const knex = require("knex");
+const knexConfig = require("./knexfile.js");
+const db = knex(knexConfig.development);
+const server = express();
+
+server.use(express.json());
+
+server.post("/projects", (req, res) => {
+  const project = req.body;
+  db("projects")
+    .insert(project)
+    .then(ids => {
+      res.status(201).json(ids);
+    })
+    .catch(err => {
+      res.status(500).json({ error: "could not add a new project", err });
+    });
+});
+
+server.post("/actions", (req, res) => {
+  const action = req.body;
+  db("actions")
+    .insert(action)
+    .then(ids => {
+      res.status(201).json(ids);
+    })
+    .catch(err => {
+      res.status(500).json({ error: "could not add a new action", err });
```

Comment:
Your two posts' callbacks look super similar. Maybe we could abstract them away in order to repeat less code? For example:
```javascript
const handlePost = table => {
  return (req, res) => {
    db(table)
      .insert(req.body)
      .then(ids => res.status(201).json(ids)
      .catch(err => res.status(500)
        .json({ error: `could not add to ${table}`, err })
      );
  };
};
```
and then you could use it like this:
```javascript
server.post('/projects', handlePost('projects'));
server.post('/actions', handlePost('actions'));
```